administrator/components/com_associations/views/association/tmpl/edit.php

```diff
@@ -63,35 +203,48 @@ function triggerSave() {
 		height: 1500px;
 		border: 0 !important;
 	}
+
+	.reference-side h3 {
+		float: left;
+		width: 50%;
+	}
 ');
+
+$input      = $this->app->input;
+$layout     = $input->get('layout', '', 'string');
+$aComponent = $input->get('acomponent', '', 'string');
+$aView      = $input->get('aview', '', 'string');
+$fLanguage  = $input->get('forcedlanguage', '', 'string');
+
 ?>
-<button id="toogle-left-panel">Show/Hide Reference (PoC)</button>
+<button id="toogle-left-panel" class="btn btn-small">Show/Hide Reference (PoC)</button>
 
-<form action="<?php echo JRoute::_('index.php?option=com_associations&view=association'); ?>"
- method="post" name="adminForm" id="item-form" class="form-validate">
+<form action="<?php echo JRoute::_('index.php?option=com_associations&view=association&layout=' . $layout . '&acomponent=' 
+		. $aComponent . '&aview=' . $aView . '&forcedlanguage=' . $fLanguage . '&id='
+		. $this->referenceID); ?>""
+ method="post" name="adminForm" id="adminForm" class="form-validate">
```

Comment:
I've added the URL parameters as @andrepereiradasilva suggested, one thing here, the &forcedlanguage parameter is not being used anywhere, @infograf768 I think it was you who suggested this parameter, is it really needed?